src/vs/workbench/contrib/terminal/browser/terminalView.ts

```diff
@@ -448,13 +475,15 @@ class SingleTerminalTabActionViewItem extends MenuEntryActionViewItem {
 	}
 }
 
-function getSingleTabLabel(instance: ITerminalInstance | null, icon?: ThemeIcon) {
```

Comment:
Removing icon here may break alt+hover on the single tab switching to the split icon?